ext/drmgr/drmgr.h

```diff
@@ -1093,6 +1111,8 @@ DR_EXPORT
  * Unregister a callback function, which takes user data as a parameter
  * for the module load event.  \return true if unregistration is successful
  * and false if it is not (e.g., \p func was not registered).
+ *
+ * See also #drmgr_unregister_module_load_event().
```

Comment:
Extraneous `#`?  I don't think it hurts though: there's no literal `#` in the resulting html, right, since that's how you link a non-function anyway.  Just odd that you've got it here only.

Reply:
Ah, missed it! I  initially included # for all functions, but then decided to remove them. Sorry about this.

Reply:
Done.